CONTRIBUTING.md

```diff
@@ -155,7 +155,7 @@ Follow the following steps if you would like to make a new pull request:
    ```bash
    git pull [--rebase] upstream <dev-branch>
    ```
-   
+
```

Comment:
Please do not add changes to unrelated files in the PR.

Reply:
Wasn't this a result of the pre-commit hook? I admit I never used the pre-commit hooks.

Reply:
pre-commit hooks are run only on the files that were changed, unless you run them explicitly on all files with `pre-commit run --all`. Anyhow, I have removed all trailing whitespace now in all source files so an easy way out is to rebase this PR on the current `develop`.

Reply:
Yep  my bad,  I had run pre-commit run --all-files, and it had autofixed all the files. I've reverted these and rebased as well to avoid merge conflicts.